### PR TITLE
Updated the day view x-axis labels to help think about engagement cycles

### LINKED ISSUE #152: Update the day view x-axis labels to help think about engagement cycles
The labels used on the 1 day view of the traffic graphs make it a bit hard for me to read some of the information I want from them. They're great for reading the number of messages received today, and from each operator, but not for thinking about weekly engagement cycles.

In the first instance, I'd suggest adding the date to every day (not just every 5), and an initial for the day of the week. Will happily take other suggestions, especially if that approach turns out to be too cluttered.

### PR DESCRIPTION
**Pull request by DanielMwendwa**
engagement_cycles
Fixes: #152

### CODE REVIEW

File: public/graph.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ class GraphController {
         let isYLimitReceivedManuallySet = false,
             isYLimitSentManuallySet = false,
             dayDateFormat = d3.timeFormat("%Y-%m-%d"),
```
Comment (IsaackMwenda) [inline public/graph.js:18]: Lets format this and use it instead of creating a new variable

File: public/graph.js [hunk 2]
```diff
@@ -128,7 +129,7 @@ class GraphController {
             sentDataStackedDaily = stackSentDaily(dailySentTotal);
 
         //Create margins for the three graphs
-        const Margin = { top: 40, right: 100, bottom: 90, left: 70 },
+        const Margin = { top: 40, right: 100, bottom: 105, left: 70 },
```
Comment (IsaackMwenda) [inline public/graph.js]: The bars seem more misaligned now, please retest and readjust.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
Requesting changes because of the misaligned bars.

File: public/graph.js (hunk 1 above)
Comment (DanielMwendwa) [reply public/graph.js:18]: ` data.forEach(function(d) {
            d.datetime = new Date(d.datetime);
            d.day = dayDateFormat(new Date(d.datetime));`
due to this if we format the first variable we'll get the error below
err

Comment (IsaackMwenda) [reply public/graph.js:18]: Ahh i see thanks.

File: public/graph.js [hunk 3]
```diff
@@ -789,13 +790,13 @@ class GraphController {
                 d3.select("#lastUpdated").classed("text-stale-info alert alert-stale-info", false);
             }
         }
-        
+
```
Comment (IsaackMwenda) [inline public/graph.js]:
```suggestion
```

File: public/graph.js [hunk 4]
```diff
@@ -789,13 +790,13 @@ class GraphController {
...
         if (GraphController.lastUpdateTimer) {
             clearInterval(GraphController.lastUpdateTimer);
         }
         GraphController.lastUpdateTimer = setInterval(setLastUpdatedAlert, 1000);
     }
-    
+
```
Comment (IsaackMwenda) [inline public/graph.js]:
```suggestion
```

**Review by IsaackMwenda — APPROVED** [review]
Thanks

**Pull request merged**